test_driver/cloud_firestore_behaviors.dart

```diff
@@ -166,11 +266,11 @@ void main() {
 
       final DocumentReference doc = messages.document();
       // updateData requires an existing document
-      await doc.setData({'foo': 'bar'});
-
-      await doc.updateData({
-        'nested.data.message': 'old value1',
-        'nested.data.unaffected_field': 'old value2',
+      await doc.setData({
```

Comment:
When I wrote the old code, I thought nested document is only for `updateData`. Actually `setData` can take nested documents.